packages/esm-reports-app/src/components/reports.scss

```diff
@@ -88,3 +88,75 @@
 .notScheduledStatusText {
   color: red;
 }
+
+.filterForm {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  gap: 1rem;
+  margin-bottom: 1rem;
+  padding: 1rem;
+  background-color: #f4f4f4;
+  border-radius: 4px;
```

Comment:
@denniskigen I'd appreciate some input on this. Usually, my rule is anything other than `1px` borders should be rems, but border-radius might be another point where it makes sense to use `px`?